agg.pretty/ADP2389ACPZ-R7.kicad_mod

```diff
@@ -0,0 +1,63 @@
+(module ADP2389 (layer F.Cu) (tedit 5A7CD2A7)
+  (fp_text reference REF** (at 0 -3.7) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_text value ADP2389 (at 0 3.8) (layer F.Fab)
+    (effects (font (size 1 1) (thickness 0.15)))
+  )
+  (fp_poly (pts (xy -1.6 0.9) (xy -1.6 0.5) (xy 1.3 0.5) (xy 1.3 0.9)) (layer F.Paste) (width 0.15))
+  (fp_poly (pts (xy -1.6 1.6) (xy -1.6 1.2) (xy 1.3 1.2) (xy 1.3 1.6)) (layer F.Paste) (width 0.15))
+  (fp_poly (pts (xy 1.6 -1.6) (xy 0.2 -1.6) (xy 0.2 -0.4) (xy 1.6 -0.4)) (layer F.Paste) (width 0.15))
+  (fp_poly (pts (xy -1.6 -1.6) (xy -0.2 -1.6) (xy -0.2 -0.4) (xy -1.6 -0.4)) (layer F.Paste) (width 0.15))
+  (fp_line (start -2.1 -2.5) (end -2.5 -2.1) (layer F.SilkS) (width 0.15))
+  (fp_line (start -2.5 2.5) (end -2.1 2.5) (layer F.SilkS) (width 0.15))
+  (fp_line (start -2.5 2.1) (end -2.5 2.5) (layer F.SilkS) (width 0.15))
+  (fp_line (start 2.5 2.5) (end 2.1 2.5) (layer F.SilkS) (width 0.15))
+  (fp_line (start 2.5 2.4) (end 2.5 2.5) (layer F.SilkS) (width 0.15))
+  (fp_line (start 2.5 2.1) (end 2.5 2.4) (layer F.SilkS) (width 0.15))
+  (fp_line (start 2.5 -2.5) (end 2.1 -2.5) (layer F.SilkS) (width 0.15))
+  (fp_line (start 2.5 -2.5) (end 2.5 -2.1) (layer F.SilkS) (width 0.15))
+  (fp_line (start -2.7 2.7) (end 2.7 2.7) (layer F.CrtYd) (width 0.01))
+  (fp_line (start -2.7 -2.7) (end -2.7 2.7) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 2.7 -2.7) (end -2.7 -2.7) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 2.7 2.7) (end 2.7 -2.7) (layer F.CrtYd) (width 0.01))
+  (fp_circle (center -2.1 -2.1) (end -1.9 -2) (layer F.Fab) (width 0.01))
+  (fp_line (start 2.5 2.5) (end 2.5 -2.5) (layer F.Fab) (width 0.01))
+  (fp_line (start -2.5 2.5) (end 2.5 2.5) (layer F.Fab) (width 0.01))
+  (fp_line (start -2.5 -2.5) (end -2.5 2.5) (layer F.Fab) (width 0.01))
+  (fp_line (start -2.5 -2.5) (end 2.5 -2.5) (layer F.Fab) (width 0.01))
+  (pad 34 smd rect (at -0.154 1.025 180) (size 3.34 1.625) (layers F.Cu F.Mask))
+  (pad 33 smd rect (at 0 -1.0375 180) (size 3.7 1.65) (layers F.Cu F.Mask))
```

Comment:
Pads 33 and 34 still don't quite seem to be in the right position. Datasheet indicates there should be 0.625mm from bottom of package to bottom edge of bottom pad, and 0.65mm from top of package to top edge of top pad, which isn't met here. Other dimensions seem OK now.